jobeet/src/jobeet/MyBundle/Tests/Provider/ProviderTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace jobeet\MyBundle\Tests\Provider;
+
+use jobeet\MyBundle\Entity\Category;
+use jobeet\MyBundle\Entity\Job;
+use jobeet\MyBundle\Manager\Manager;
+use jobeet\MyBundle\Provider\Provider;
+
+class ProviderTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @return \Traversable
+     */
+    private function getGenericJobs()
+    {
+        $genericCategory = new Category();
+        $genericCategory->setName('Generic Category');
+
+        for ($i = 0; $i < 10; $i++) {
+            $job = new Job();
```

Comment:
ez is egy metódus, vagy ne adj isten egy trait, vagy bármi, de nem lemásolva ugyanaz.